src/main.js

```diff
@@ -0,0 +1,28 @@
+
+import fs from 'fs';
+import { getRepositories } from './githubApi.js';
+import { summarizeRepositories , filterRepositoriesWithMoreThanFiveStars, getLastFiveUpdatedRepositories , sumOfAllRepositoryStars} from './repositoryUtils.js';
+
+const TOKEN = process.env.GITHUB_TOKEN;
+
+    async function main() {
+        try {
+            const repositoriesPromise = getRepositories(TOKEN);
+            const repositories = await repositoriesPromise;
```

Comment:
```suggestion
            const repositories = await getRepositories(TOKEN);
```

Reply:
Prettier added with yarn
I have a problem with the rule of a single quote that is not being applied 
Suggested change done